packages/roosterjs-content-model/lib/modelApi/image/applyImageBorderFormat.ts

```diff
@@ -0,0 +1,50 @@
+import { Border, extractBorderValues } from '../../domUtils/borderValues';
+import { ContentModelImage } from '../../publicTypes/segment/ContentModelImage';
+import { ptToPx } from '../../formatHandlers/utils/pointsToPixels';
+
+/**
+ * @internal
+ */
+export default function applyImageBorderFormat(
+    image: ContentModelImage,
+    border: Border,
+    isPt?: boolean
```

Comment:
Can we just parse the unit from the value automatically? We have a util parseValueWithUnit to do this